README.rst

```diff
@@ -1046,6 +1046,15 @@ Choose from the list of available rules:
     ``['assertAttributeEquals', 'assertAttributeNotEquals', 'assertEquals',
     'assertNotEquals']``
 
+* **php_unit_test_annotation**
+
+  Adds or removes @test annotations from tests, following configuration.
+
+  Configuration options:
+
+  - ``annotation`` (``false``, ``true``): whether to use the @test annotation or not;
```

Comment:
Quick heads-up, in case you wondered, `README.rst` will eventually need to be regenerated by running 

```
$ ./php-cs-fixer readme > README.rst
```

Reply:
Ah thanks, forgot about that